nemo_skills/code_execution/sandbox.py

```diff
@@ -246,6 +247,9 @@ def strip_ansi_codes(text):
     }}
 print(json.dumps(to_return))
 """
+        elif language in ["python", "pypy3", "python3"]:
+            TO_EXECUTE = generated_code
```

Comment:
Should we raise an error here if session_id is not None, similar to Lean? As current logic doesn't reuse previous code snippets, right?